quicksoinc/slate-plugins-next
# Make trigger work with `+` sign

Issue:

## What I did

Changed the trigger to make it work with custom chars like `+` . The `word` is not always a just chars, it can start with ` ` (space) like ` foo` so the regexp will capture this from now.

## How to test

- Is this testable with Jest?
- Does this need a new example in Storybook?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in
your PR.plugin
